addons/website/views/assets.xml

```diff
@@ -91,6 +91,8 @@
         <script type="text/javascript" src="/website/static/src/js/post_link.js"/>
         <!-- Custom empty file for user javascript -->
         <script type="text/javascript" src="/website/static/src/js/user_custom_javascript.js"/>
+        
+        <script type="text/javascript" src="/website/static/src/js/widgets/fullscreen_indication.js"/>
```

Comment:
You've put this script under the `Custom empty file for user javascript` comment, not a big deal but not as clear as it could be, since it's only a dependency for the website root, you should probably put it bellow the block of scripts that start with the website_root one.